typed_sql/lib/src/adapter/adapter.dart

```diff
@@ -195,7 +192,7 @@ abstract base class DatabaseAdapter extends Executor {
   /// > compatibility guarantees.
   ///
   /// This is intended for debugging purposes, and pairs nicely with
-  /// [printOnFailure] from `package:test/test.dart`.
+  /// `printOnFailure` from `package:test/test.dart`.
```

Comment:
Did the docimport not work?

Reply:
On pub.dev we don't resolve `dev_dependencies`, so analysis will fail, but also rendered dartdoc won't have the links.

Reply:
Ah yeah - didn't think about that...